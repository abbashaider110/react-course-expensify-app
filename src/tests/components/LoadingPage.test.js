import {shallow} from 'enzyme';
import React from 'react';
import LoadingPage from '../../components/LoadingPage';

test('snapshot for laoding page', () =>{
    const wrapper = shallow(<LoadingPage/>);
    expect(wrapper).toMatchSnapshot();
})