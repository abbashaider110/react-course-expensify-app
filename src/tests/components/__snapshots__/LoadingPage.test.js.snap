// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for laoding page 1`] = `
<div
  className="loader"
>
  <img
    className="laoder__image"
    src="/images/loader.gif"
  />
</div>
`;
